src/Options/OptionsDetails.hpp

```diff
@@ -160,10 +160,10 @@ struct has_upper_bound_on_size<
 
 template <typename Group, typename OptionList, typename = std::nullptr_t>
 struct print_impl {
-  static std::string apply(const int max_label_size) noexcept {
```

Comment:
Would it be better to completely remove max_label_size as an argument to print_impl::apply here and below, and in `struct print` below, and in line 510 of ParseOptions.hpp?   As far as I can tell, after your changes the argument is never used.

Reply:
That seems reasonable to me. @wthrowe since you wrote this code, how do you feel about removing `max_label_size`?